float64 rollangle
float64 pitchangle
float64 yawangle

# strait down to table
# parallel gripper direction with the arm base
float64 DEFAULT_ROLL = 3.14159265358979323846
float64 DEFAULT_PITCH = 0.0
float64 DEFAULT_YAW = -3.14159265358979323846

float64 to_x
float64 to_y
float64 to_z

float64 vel
---
uint32 retval

# error code
uint32 OK = 0           # OK
uint32 NO_IK = 1        # no IK solution found
uint32 TURN_BACK = 2    # IK solutions require turning backwards
uint32 LOW_GRIPPER = 6  # move gripper into ground
uint32 OTHER = 8        # other unknown error

